dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.cpp

```diff
@@ -2375,7 +2317,7 @@ RtpsUdpDataLink::bundle_mapped_meta_submessages(AddrDestMetaSubmessageMap& adr_m
     for (DestMetaSubmessageMap::iterator dest_it = addr_it->second.begin(); dest_it != addr_it->second.end(); ++dest_it) {
       for (MetaSubmessageIterVec::iterator resp_it = dest_it->second.begin(); resp_it != dest_it->second.end(); ++resp_it) {
         // Check before every meta_submessage to see if we need to prefix a INFO_DST
-        if (dest_it->first != GUID_UNKNOWN && dest_it->first != prev_dst) {
+        if (dest_it->first != prev_dst) {
```

Comment:
Why was this reverted?

Reply:
It wasn't, this is a different line of code from the one you changed in
send_bundled_submessages. The bundling logic ought to match the sending
logic, and since you updated the sending logic for this check, I updated
the bundling logic to match.

Reply:
Got it.